MarkBind/markbind
# Optimize code printing

**What is the purpose of this pull request?**


Resolves #2438 



**Overview of changes:**
Add CSS which 
- Makes code light theme automatically when printing 
- Removes code wrap and code copy buttons when printing 
- Include code highlighting when printing 

**Anything you'd like to highlight/discuss:**
If anyone can help to test on Mac, that will be great. 

There seems to be some issue with printing on Edge.. the CSS isnt applied on the preview even though the emulator reflects the changes.. 
TLDR: Don;t use Edge to print

**Testing instructions:**
Make code theme dark 

```js {start-from=6 .line-numbers highlight-lines="7, 9"}
function add(a, b) {
    const sum = a + b;
    console.log(`${a} + ${b} = ${sum}`);
    return sum;
}
```

Notice that code wrap and code copy buttons are gone
Code highlight is printed 
Code theme is light



**Proposed commit message: (wrap lines at 72 characters)**
Optimize code blocks for printing



---

**Checklist:** :ballot_box_with_check:






---

**Reviewer checklist:**

Indicate the [SEMVER](https://semver.org/) impact of the PR:

At the end of the review, please label the PR with the appropriate label: `r.Major`, `r.Minor`, `r.Patch`.

Breaking change release note preparation (if applicable):
- To be included in the release note for any feature that is made obsolete/breaking

> Give a brief explanation note about:
>   - what was the old feature that was made obsolete
>   - any replacement feature (if any), and
>   - how the author should modify his website to migrate from the old feature to the replacement feature (if possible).
